lib/zold/commands/fetch.rb

```diff
@@ -124,5 +128,11 @@ def age(json)
         "#{(sec / 3600).round}h"
       end
     end
+
+    def enough_trust?(total, trust_value)
```

Comment:
@mgrassotti please add a puzzle to DRY it later or extract somewhere. The same method exists in FETCH and PUSH.